alkos/kernel/arch/x86_64/enables/enable_extensions.nasm

```diff
@@ -0,0 +1,101 @@
+    bits 64
+
+    section .text
+    global enable_extensions
```

Comment:
I would much rather call a set of functions one by one than just one big one that enables "extensions".
1. I didn't understand what you enable from the name
2. You want granularity - so for example if you check if a feature is there in the cpuid, and its not there, give me an error code and print that this specific feature is not there in the cpu via vga. My call is mimick boot32 and remain consistent

Reply:
Comment on unfinished PR, closing.